sdcard/rootfs/kube-systemd/usr/lib/systemd/system/k8s-master.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=The Master Components for Kubernetes
-After=docker.service
+Requires=docker.service
```

Comment:
I don't think it's too serious, but please leave as-is:
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Requires=
> If a unit foo.service requires a unit bar.service as configured with Requires= and no ordering is configured with After= or Before=, then both units will be started simultaneously and without any delay between them if foo.service is activated. Often, it is a better choice to use Wants= instead of Requires= in order to achieve a system that is more robust when dealing with failing services.